avo-hq/avo
# fix: card generators broken; added tests; show avo version

# Description


This outputs Avo's version number in the console whenever a user uses the Avo generators.

The PR also fixes an issue with card generators and adds tests for them.

# Checklist:



## Manual review steps


1. Run any generator
2. See the avo version and license type

-

1. Run `bin/rails generate avo:card:partial` and see the card being generated.
2. Try with `metric` and `chartkick` too.

Manual reviewer: please leave a comment with output from the test if that's the case.
